src/test/java/pl/project13/maven/git/GitDirLocatorTest.java

```diff
@@ -52,7 +52,9 @@ public void shouldUseTheManuallySpecifiedDirectory() throws Exception {
       assert foundDirectory != null;
       assertThat(foundDirectory.getAbsolutePath()).isEqualTo(dotGitDir.getAbsolutePath());
     } finally {
-      dotGitDir.delete();
+      if (!dotGitDir.delete()) {
+        dotGitDir.deleteOnExit();
+      }
```

Comment:
I don't know - both ways seem acceptable: deleteOnExit is in tests (not such a critical piece of code). On the other hand - it's inside "target", so clean`ed anyway. In addition, randomization makes deleteOnExit even less needed for test repetition without cleaning.